src/views/check/could-not-match-national-insurance.njk

```diff
@@ -7,13 +7,18 @@
 {% set gtmJourney = "check - retry" %}
 
 {% block mainContent %}
+    <h1 class="govuk-heading-l" id="header" data-page="{{ googleTagManagerPageId }}"
+        xmlns="http://www.w3.org/1999/html">{{ 'could-not-match-national-insurance.h1' | translate }}</h1>
+    <p class="govuk-body" >{{'could-not-match-national-insurance.paragraph1' | translate }}</p>
+
+    <p class="govuk-body" >{{'could-not-match-national-insurance.paragraph2' | translate }}</p>
+
   {% call hmpoForm(ctx) %}
       {{ hmpoRadios(ctx, {
             id: "retryNationalInsuranceRadio",
             namePrefix: "retryNationalInsuranceRadio",
             fieldset: {
                 legend: {
-                    isPageHeading: true,
```

Comment:
Now that we've removed the override we can actually probably not rely on the defaults from the form wizard which means the `block mainContent` can be removed and the `paragraph1` `paragraph2` stuff can be changed back to `content`? Or am I missing something?